.github/workflows/release-workflow.yml

```diff
@@ -131,7 +133,7 @@ jobs:
           name: linux-artifacts
           path: |
             nexus-staging/
-            samm-cli-linux.tar.gz
+            samm-cli-${{ github.event.inputs.release_version }}-linux-x86_64.tar.gz
             samm-cli-*.jar
```

Comment:
Please replace this line with the specific file names, so that we do not include the "samm-cli-${release_version}-for-native-image.jar" any more:

```
samm-cli-${release_version}-javadoc.jar
samm-cli-${release_version}-sources.jar
samm-cli-${release_version}-tests.jar
```